11/main.js

```diff
@@ -0,0 +1,294 @@
+const russiaIMGUrl =
+  "https://img2.joyreactor.cc/pics/post/Fantasy-art-красивые-картинки-ork-2468149.jpeg";
+const USAIMGRUrl =
+  "https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg";
+const UaIMGUrl =
+  "https://upload.wikimedia.org/wikipedia/commons/thumb/a/ad/Flag_of_Ukraine_%28with_coat_of_arms_2%29.svg/2560px-Flag_of_Ukraine_%28with_coat_of_arms_2%29.svg.png";
+const ItaliaIMG =
+  "https://upload.wikimedia.org/wikipedia/commons/thumb/0/03/Flag_of_Italy.svg/1200px-Flag_of_Italy.svg.png";
+const UKIMG =
+  "https://upload.wikimedia.org/wikipedia/en/thumb/a/ae/Flag_of_the_United_Kingdom.svg/1200px-Flag_of_the_United_Kingdom.svg.png";
+const FranciaIMG =
```

Comment:
It's better to start names with regular letter 
```suggestion
const franceIMG =
```